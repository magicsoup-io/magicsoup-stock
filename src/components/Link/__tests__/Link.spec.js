import React from 'react'
import renderer from 'react-test-renderer'

import { Link } from '../../../index'

it('renders <Link /> without throwing', () => {
  const tree = renderer.create(
    <Link href="https://www.google.com">A basic link element</Link>,
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
