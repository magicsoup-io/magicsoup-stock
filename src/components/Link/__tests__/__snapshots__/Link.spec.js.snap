// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Link /> without throwing 1`] = `
<a
  className="sc-bdVaJa sc-bwzfXH jhKGfG"
  color="blue"
  href="https://www.google.com"
>
  A basic link element
</a>
`;
